python/ray/data/_internal/block_batching/iter_batches.py

```diff
@@ -34,6 +34,7 @@ def iter_batches(
     shuffle_seed: Optional[int] = None,
     ensure_copy: bool = False,
     prefetch_batches: int = 1,
+    gpu_prefetch_batches: int = 1,
```

Comment:
Shouldn't this be for iter_torch_batches() only?

Reply:
`DataIterator.iter_torch_batches()` calls `DataIterator.iter_batches()`, which calls this `iter_batches` function in `block_batching/iter_batches.py`